apis/addons/v1alpha1/addons_types.go

```diff
@@ -231,6 +231,10 @@ type AddonNamespace struct {
 	// Name of the KubernetesNamespace.
 	// +kubebuilder:validation:MinLength=1
 	Name string `json:"name"`
+
+	Labels map[string]string `json:"labels"`
```

Comment:
These should be optional right? (we need the omit-empty tag as well)

Reply:
whoops! Nice catch.